.github/workflows/black-check.yml

```diff
@@ -0,0 +1,26 @@
+# For more information on how to use this template please refer to:
+# http://tardis-sn.github.io/tardis/development/continuous_integration.html
+
+name: black
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install Black
+        run: pip install black
+
+      - name: Run Black
+        run: black --check tardis
```

Comment:
Looks very good! Please make sure that this command is taking black configuration file we have at tardis in account. If it is not, then you can also explicitly point it out by `--config` flag. Details here: https://black.readthedocs.io/en/stable/pyproject_toml.html#where-black-looks-for-the-file

Reply:
Yes, it taking config from `pyproject.toml` just right.